### PR TITLE
Log frequency information in CSV file

### PR DESCRIPTION
**Pull request by aferoz21**
1) Log the tensile run frequency information in CSV file. 
2) Rearrange the total flops information display to the last column in CSV file.

### CODE REVIEW

File: Tensile/LibraryLogic.py [hunk 1]
```diff
@@ -461,10 +461,21 @@ def addFromCSV(self, dataFileName, numSolutions, solutionMap):
 
         # get problem size
         problemSize = []
+        print1("# Feroz exactProblemSizes: %s" % (self.exactProblemSizes))
```
Comment (nakajee) [inline Tensile/LibraryLogic.py:464]: Do you really need this print?

File: Tensile/Source/client/include/ResultFileReporter.hpp [hunk 2]
```diff
@@ -79,6 +79,9 @@ namespace Tensile
             int64_t     m_winnerSolutionIdx = -1;
             double      m_fastestGflops     = -1.0;
             double      m_fasterTimeUS      = -1.0;
+            // for logging winner GFx clock in CSV file
+            uint16_t m_currGfxClock   = 0;
```
Comment (nakajee) [inline Tensile/Source/client/include/ResultFileReporter.hpp:83]: Could you keep the alignment?

Comment (aferoz21) [reply Tensile/Source/client/include/ResultFileReporter.hpp]: clang-format did that alignment, I  modified to match the exisiting code.

File: Tensile/LibraryLogic.py (hunk 1 above)
Comment (aferoz21) [reply Tensile/LibraryLogic.py]: I removed the debug statements.

File: Tensile/Source/client/include/ResultFileReporter.hpp (hunk 2 above)
Comment (nakajee) [reply Tensile/Source/client/include/ResultFileReporter.hpp]: > clang-format did that alignment, I modified to match the exisiting code.

OK. Then, please follow clang-format.

**Comment by nakajee** [comment]
Why some of CI tests failed?

**Comment by aferoz21** [comment]
> Why some of CI tests failed?
We are swapping the order of lda,ldb,ldc,ldd format in CSV file.  There are few GEMM tests does not have batch count in the problem size. so the error comes due to the incorrect index order.  I have to cater for format 2. 
768, 768, 1, 16, 768, 768, 172032, 768 -> Format 1
768, 768, 16, 768, 768, 172032, 768  -> Format 2

File: Tensile/LibraryLogic.py [hunk 3]
```diff
@@ -463,8 +463,17 @@ def addFromCSV(self, dataFileName, numSolutions, solutionMap):
...
+        #but Tensile problem size format size is [M,N,Batchcount,K,lddC,lddD,lddA,lddB]
+        #so we are swapping the ldc,ldd position similar to tensile format
+        
+        #take backup of lda,ldb position.
+        ldaldbBackup = problemSize[4], problemSize[5]
+        #Move the ldc,ldd postion to 4th and 5th position
+        problemSize[4], problemSize[5] = problemSize[6], problemSize[7]
+        #Move the lda,ldb postion to the end (ie 6th and 7th position)
+        problemSize[6], problemSize[7] = ldaldbBackup
         problemSize = tuple(problemSize)
```
Comment (babakpst) [inline Tensile/LibraryLogic.py:476]: Please revert this change.

File: Tensile/Source/client/source/ResultFileReporter.cpp [hunk 4]
```diff
@@ -147,18 +152,19 @@ namespace Tensile
...
                 // Values for these come separately.
-                m_output.setHeaderForKey(ResultKey::LDD, "LDD");
-                m_output.setHeaderForKey(ResultKey::LDC, "LDC");
                 m_output.setHeaderForKey(ResultKey::LDA, "LDA");
                 m_output.setHeaderForKey(ResultKey::LDB, "LDB");
-                m_output.setHeaderForKey(ResultKey::TotalFlops, "TotalFlops");
+                m_output.setHeaderForKey(ResultKey::LDC, "LDC");
+                m_output.setHeaderForKey(ResultKey::LDD, "LDD");
+                m_output.setHeaderForKey(ResultKey::GfxFrequency, "WinnerFreq");
                 if(m_extraCol)
```
Comment (babakpst) [inline Tensile/Source/client/source/ResultFileReporter.cpp:160]: Let's move WinnerFreq before TotalFlops.

**Review by babakpst — APPROVED** [review]
